uk.co.compucorp.civicrm.tasksassignments/api/v3/Document.php

```diff
@@ -232,32 +214,43 @@ function civicrm_api3_document_create_multiple($params) {
  */
 function _civicrm_api3_document_create_spec(&$params) {
 
-  //default for source_contact_id = currently logged in user
-  $params['source_contact_id']['api.default'] = 'user_contact_id';
```

Comment:
Do we really need to remove this? I know civi defaults to current user on the API explorer for most of its API's that uses a contact ID.